.travis.yml

```diff
@@ -15,31 +11,46 @@ branches:
 env:
   global:
     - RENDERDOC_TRAVIS_BUILD=1 RENDERDOC_CI_BUILD=1
-  matrix:
-    - CODE_BUILD=1 LINUX_BUILD=1 DOCS_BUILD=0
 
 # only build docs once on linux
```

Comment:
not sure this comment makes sense anymore - also docs is disabled on linux right now